okcupyd/json_search.py

```diff
@@ -26,7 +27,11 @@ def SearchFetchable(session=None, **kwargs):
 
     :param session: A logged in session.
     :type session: :class:`~okcupyd.session.Session`
+    :param order_by: Expected values: 'match', 'online', 'special_blend'
     """
+##:param location: A location string which will be used to filter results.
```

Comment:
I wasn't meaning to be following any markup syntax with these. I was writing this code more with a view to it's being reviewed by you than to its making its way directly into the codebase. I hope it's a useful effort, but I was being irresponsible in giving it this much time right now anyway, and won't be able to put much more time into it until the end of the month. You should just take what you want and edit it as you please. If there are changes I can quickly push that'd make it easier for you to import I'm glad to help with them.